### ISSUE TITLE
In OperatingReport, Clarify what each EnergyAC value represents. 

### ISSUE THREAD

**Issue by kevinkirkl3y** [Open-Orange-Button/Orange-Button-Taxonomy#210]
We need to clarify various instances of EnergyAC values that exist within OperatingReport and what they will represent. May need to add Usage Tips.

**Comment by kevinkirkl3y** [comment]
This could be a shortfall of the Editor itself. Doesn't allow users to add clarification to a specific use of an element or object. The definitions and usage tips will be applied generally to all instances of an element in the Taxonomy.

**Comment by kevinkirkl3y** [comment]
@cwhanse It looks like you have addressed this in your Github page. Do you see any development outside the pages that needs to be done to address this issue?

**Comment by cwhanse** [comment]
Yes, https://github.com/Open-Orange-Button/Orange-Button-Editor/issues/29 and #215 could help. I don't think we want context-specific Documentation strings; an element should mean one and only one thing. But adding Documentation to Arrays would help, as would adding Usage Tips to the Objects.
